packages/schema/src/schema/jcad.json

```diff
@@ -18,6 +18,15 @@
         }
       },
       "additionalProperties": false
+    },
+    "outputs": {
```

Comment:
We may want to provide a way to clean outputs in the future?

Reply:
indeed, adding it now, the output needs to be deleted when the operator is removed